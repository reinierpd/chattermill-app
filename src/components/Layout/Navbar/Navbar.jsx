import React from 'react';
import AppBar from '@material-ui/core/AppBar';
import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';
import { Link } from 'routes';

/**
 * @description
 * React hook Navbar
 *
 * @returns {*} - Image as link.
 */
const Navbar = () => (
  <AppBar position="static">
    <Toolbar>
      <Typography variant="h5" color="inherit">
        <Link route="dashboard">
          <a style={{ textDecoration: 'none', color: 'inherit' }}>
            Chattermill
          </a>
        </Link>
      </Typography>
    </Toolbar>
  </AppBar>
);

export default Navbar;
