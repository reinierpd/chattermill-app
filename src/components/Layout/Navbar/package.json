{
  "name": "navbar",
  "version": "0.0.1",
  "private": true,
  "main": "Navbar.jsx"
}
