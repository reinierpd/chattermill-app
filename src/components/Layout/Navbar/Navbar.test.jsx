/* eslint-env jest */
/* eslint-disable react/jsx-filename-extension */
import React from 'react';
import renderer from 'react-test-renderer';
import Navbar from './Navbar';

describe('Navbar Component', () => {
  it('render layout', () => {
    const wrapper = renderer.create(<Navbar />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
