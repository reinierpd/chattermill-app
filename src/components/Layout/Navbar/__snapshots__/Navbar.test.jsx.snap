// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Component render layout 1`] = `
<header
  className="MuiPaper-root MuiPaper-elevation4 MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary"
>
  <div
    className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
  >
    <h5
      className="MuiTypography-root MuiTypography-h5 MuiTypography-colorInherit"
    >
      <a
        href="/"
        onClick={[Function]}
        style={
          Object {
            "color": "inherit",
            "text-decoration": "none",
          }
        }
      >
        Chattermill
      </a>
    </h5>
  </div>
</header>
`;
